ffi/sfml-build/src/lib.rs

```diff
@@ -26,12 +26,16 @@ pub fn link_csfml(lib_name: &str) {
 
 // Add search path for SFML library files
 pub fn link_sfml(lib_name: &str) {
-    // SFML_HOME points to the base SFML directory
-    // Let cargo find the SFML library files there
-    if let Ok(sfml_home) = var("SFML_HOME") {
-        println!("cargo:rustc-link-search=native={}/lib", sfml_home);
-    }
+    // The windows build of CSFML links SFML libraries statically, so no
+    // need to link them to the rust executable as well.
+    if !cfg!(target_family = "windows") {
```

Comment:
Before I forget: `cfg!` is actually not the correct way to do target-specific configuration, because `cfg!` is derived from the host, not the target. Cargo provides [`CARGO_CFG_`](http://doc.crates.io/environment-variables.html#environment-variables-cargo-sets-for-build-scripts) variants for build scripts.

Reply:
To clarify: This is probably fine if you're only ever building rust-sfml for the same target as the host, but it messes up cross-compilation.

Reply:
So something like `var("CARGO_CFG_TARGET_OS").unwrap_or_default().as_str() != "windows"` ?